### DISCUSSION TITLE
[dropdown] the dropdown made by a 'select' tag isn't shown as the fomantic website shows it

### DISCUSSION THREAD

**Discussion by asantech** [fomantic/Fomantic-UI#1692] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report
 the dropdown made by a 'select' tag isn't shown as the fomantic website shows it

## Steps to reproduce
1. add the fomatic-ui CSS files to your project 
2. copy the select dropdown's matching HTML from the fomantic-ui website and paste it in your file
3. run the code

## Expected result
the dropdown is shown as in the fomantic-ui website

## Actual result
the dropdown is shown as a pure javascript selection dropdown

## Testcase
https://jsfiddle.net/tpv3syoL/1/

## Screenshot (if possible)
![image](https://user-images.githubusercontent.com/50336897/94990588-1623c200-058a-11eb-8b81-60972d2e3fcc.png)

## Version
2.8 (latest version)

**Comment by ko2in** [comment, marked as answer]
Using with `select` tag requires to invoke `dropdown` module. It cannot generate the dropdown rendering with DIV via CSS. A simple CSS dropdown can only build using DIV and the other HTML elements.

**Comment by ko2in** [comment]
Closed. Already explained the usage with `select` tag. If you've further issue, feel free to reopen.

**Comment by asantech** [comment]
hi how are you?

i wanted to know that:
1. in the site.variables file, if i change the @fontsize to 10px, should i also change the @emsize to 10px also, so that the modules and elements sizes dont break and have the exact ratio when @fontsize was in 14px?
2. and can you please tell how can i change the @fontsize to 10px, but also have the fomantic elements,modules and etc sizes just like as it was when @fontsize was 14px(default)?
3. on addition to question 2, can you also please tell what other @css of the site.variables file should i also change to have the fomantic element,modules and etc sizes exactly as they were before? (if you can give me all of list of @css to change, it would be great(if possible))
(for example the @csses to be changed so that the small, big, etc classes keeps their default sizes(when html tag font's size was default 14px and etc)

thank you

note: the reason i persist on @fontsize to 10px is because of the custom parts of the website (which dont use fomantic ui) and i like to convert pixel sizes to rem more easily.
and the reason that i want the fomantic elements,modules and etc to be the same as before is because

**Comment by ko2in** [comment]
Changing `@fontSize` doesn't have critical impact on FUI modules but `@emSize` does. Because, most of `rem` and `em` units are calculated upon `@emSize` value you set in your theme variables. Try searching `@fontSize` in all of FUI files (easy to do in IDE like VScode or Sublimetext), you'll see it doesn't use much and it uses for `font-size` property only.

The font-size for HTML node uses `@emSize`, so you would need to change `@emSize`, not `@fontSize`. If you changed the `@emSize` values from 14px to 10px, the calculation of `em` and `rem` units of other FUI components (such as font-size, margin, padding) which relies on `@emSize` will be changed accordingly.

Example: the following header element will have smaller font size:
```
h1.ui.header .sub.header {
  font-size: 1.1rem;
}
```
So, if you want to keep it as 14px font-size, you should increase it 1.4 times then the current size which will be: `1.54rem`.

I would suggest you to check the built-in themes such as amazon, github to learn how they've different size and you'll know what you need to customize for your theme.
